### ISSUE TITLE
Breadcrumb text doesn't have enough contrast with background

### ISSUE THREAD

**Issue by thanyabegum** [pulibrary/dpul#1217]
Text in the breadcrumb doesn't have enough contrast with its background. One solution is to make the text darker, while another is to make the background lighter.

<img width="531" alt="Screen Shot 2021-11-05 at 2 38 16 PM" src="https://user-images.githubusercontent.com/66143640/140561977-3e67ec8b-cc3b-43ce-a5e3-1951f63aaf31.png">

[Learn more about this failing Lighthouse test.](https://web.dev/color-contrast/?utm_source=lighthouse&utm_medium=devtools)
